indico/modules/designer/models/templates.py

```diff
@@ -152,6 +168,18 @@ def is_ticket(self):
     def __repr__(self):
         return format_repr(self, 'id', 'event_id', 'category_id', _text=self.title)
 
+    def link_regform(self, regform):
+        """Link this template to a registration form."""
+        self.registration_form = regform
+        db.session.commit()
```

Comment:
i'd rather not have commits in such places. we already commit at the end of a request if there's no error, and usually one does not expect some method on a model to commit early